locations/spiders/seven_eleven_ca_us.py

```
@@ -24,35 +26,43 @@
-            callback=self.parse_stores,
-        )
-
-    def parse(self, response: Response, **kwargs: Any) -> Any:  # parse token
-        self.token = re.search(r"access_token\\\":\\\"(.+?)\\\"", response.text).group(1)
-        yield self.next_request()
-
-    def parse_stores(self, response: Response, **kwargs: Any):
+    api = "https://www.7-eleven.com/api/v5/stores/search"
+    search_radius = 100
+    max_results = 1000
+    searchable_points_files = ["us_centroids_100mile_radius_state.csv", "ca_centroids_100mile_radius.csv"]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)